### PR TITLE
Refactor code to fully utilize ES6

### PR DESCRIPTION
**Pull request by 10-menachi**
Added:
- let and const keywords appropriately
- Arrow functions
- Date and time using Luxon
- Modules

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,72 +1,25 @@
...
 const form = document.querySelector('#form');
 const title = document.querySelector('#book-title');
 const author = document.querySelector('#author');
 
-const book = new Book();
-
-book.showBooks();
+const library = new Library(booksSection);
+library.books = library.getBooks();
+library.showBooks();
```
Comment (Ibizugbe) [inline index.js:15]: Excellent job refactoring your code to use `es6` functionalities👍. currently, your navigation links are not working and it is impossible to display the other pages of the application🥲. I suggest you kindly fix that👍.


https://user-images.githubusercontent.com/64643995/236780139-bf3fec41-2110-4d65-9a78-0ec9c1cd74b1.mp4

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @10-menachi 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] `ES6` arrow function and modules have been properly used👍.
- [x] Github flow have been used👍.
- [x] Linters are properly setup and they are passing👍.
- [x] Project directory is properly setup with the right files👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (10-menachi) [reply index.js:15]: Noted with thanks

Comment (10-menachi) [reply index.js:15]: /home/chrisdroid/Videos/Screencasts/Screencast from 08-05-2023 12:45:13 ALASIRI.webm

I have not changed anything but the code works as expected. Kindly assist.

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @10-menachi 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**